### PR TITLE
feat: [closes #373] add olive crop and olive oil

### LINKED ISSUE #373: Olives and olive oil
Farmhand should have an Olive crop and a related Olive Oil recipe.

### PR DESCRIPTION
**Pull request by lstebner**
### What this PR does

adds olive crop, olive oil, and garlic bread

### Additional information

Screen Shot 2023-01-10 at 5 26 21 AM

Screen Shot 2023-01-10 at 5 26 27 AM

Screen Shot 2023-01-10 at 5 26 15 AM

Screen Shot 2023-01-11 at 8 19 48 PM

### CODE REVIEW

File: src/data/recipes.js [hunk 1]
```diff
@@ -56,6 +56,20 @@ export const butter = itemify({
...
 
+/**
+ * @property farmhand.module:recipes.oliveOil
+ * @type {farmhand.recipe}
+ */
+export const oliveOil = itemify({
+  id: 'olive-oil',
+  name: 'Olive Oil',
+  ingredients: {
+    [items.olive.id]: 15,
```
Comment (jeremyckahn) [inline src/data/recipes.js:67]: Apparently it takes [an incredible amount of olives to produce olive oil](https://oliveemotion.com/en/how-many-olives-are-needed-to-make-one-liter-of-olive-oil/):

> To obtain 1 liter of extra virgin olive oil, 5 to 6 kg of olive are required, approximately 5200 to 8000 olives.

That might be a little extreme for what we're going for, but what do you think about upping the challenge here to be slightly closer to reality? Maybe ~250 olives?

File: src/data/recipes.js [hunk 2]
```diff
@@ -56,6 +56,20 @@ export const butter = itemify({
...
+  },
+  condition: state => state.itemsSold[items.olive.id] >= 150,
```
Comment (jeremyckahn) [inline src/data/recipes.js:69]: If we significantly increase the amount of olives need to produce oil, we should scale up the condition logic to be consistent with the other recipes. Perhaps requiring 400-500 olives to be sold?

**Review by jeremyckahn — COMMENTED** [review]
Great work here! I just pushed https://github.com/jeremyckahn/farmhand/commit/c7670e477498931e56e8507791413492851d98e2, which centers the `olive-growing` image:

Olive Growing preview

The code and art all look great, but I wanted to get your thoughts on tweaking the Olive Oil values before we merge.

File: src/data/recipes.js (hunk 1 above)
Comment (lstebner) [reply src/data/recipes.js:67]: oh wow, considering most recipes that use olive oil probably only need one i'm cool with upping it. it's going to be worth an insane amount based on that so we should probably also change the name to "extra virgin olive oil" haha. 

this also makes me wonder if the vegetable oil should have used more soybeans.... i've merged that but maybe we should alter it before releasing the update. thoughts?

File: src/data/recipes.js (hunk 2 above)
Comment (lstebner) [reply src/data/recipes.js:69]: yep that makes sense to me!

File: src/data/recipes.js (hunk 1 above)
Comment (jeremyckahn) [reply src/data/recipes.js:67]: These both sounds like good ideas to me! #372 is merged but not yet released, so tweaking it should be no problem.

Comment (lstebner) [reply src/data/recipes.js:67]: looks like producing vegetable oil from soybeans is even more extreme haha. 7-8kg per liter is what i found, and an average bean weighs like .15g. granted a soybean plant produces more than just one bean, it's still dozens of thousands. our soybean crop does grow quickly though, so maybe like 300 or 400? 400 sounds like a lot but it's less than fully filling the largest field twice so maybe that's ok?

Comment (lstebner) [reply src/data/recipes.js]: i split the difference and went 350 soybeans for veggie oil; and went with your suggestion of 250 for olive oil

**Comment by lstebner** [comment]
@jeremyckahn updated recipe ingredient amounts and rebased to resolve a conflict from my sweet potato branch merging. please re-review when you have time!

File: src/data/recipes.js [hunk 3]
```diff
@@ -330,9 +344,9 @@ export const vegetableOil = itemify({
   id: 'vegetable-oil',
   name: 'Vegetable Oil',
   ingredients: {
-    [items.soybean.id]: 30,
+    [items.soybean.id]: 350,
```
Comment (jeremyckahn) [inline src/data/recipes.js:347]: 💯

File: src/data/recipes.js (hunk 1 above)
Comment (jeremyckahn) [reply src/data/recipes.js]: That all seems balanced to me. Thanks for making the change!

**Review by jeremyckahn — APPROVED** [review]
Excellent work! Thanks for making all the tweaks. LGTM!

**Review by jeremyckahn — APPROVED** [review]
LGTM! Great work to everyone involved. I'll merge and ship this now!

**Pull request merged**